libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/auth/RustMatrixAuthenticationService.kt

```diff
@@ -132,11 +116,15 @@ class RustMatrixAuthenticationService @Inject constructor(
     override suspend fun setHomeserver(homeserver: String): Result<Unit> =
         withContext(coroutineDispatchers.io) {
             runCatching {
-                authService.configureHomeserver(homeserver)
-                val homeServerDetails = authService.homeserverDetails()?.map()
-                if (homeServerDetails != null) {
-                    currentHomeserver.value = homeServerDetails.copy(url = homeserver)
-                }
+                val client = getBaseClientBuilder()
+                    .serverNameOrHomeserverUrl(homeserver)
+                    .build()
+                this@RustMatrixAuthenticationService.client = client
```

Comment:
Maybe to avoid writing `this@RustMatrixAuthenticationService` 3 times, rename the `client` above to `newClient`

Reply:
This and all other similar comments should be fixed by https://github.com/element-hq/element-x-android/pull/3068/commits/b429b18e8b14e190843286851824f3ecdabdd746 🤞